tapdb/asset_minting.go

```diff
@@ -429,6 +434,71 @@ func (a *AssetMintingStore) CommitMintingBatch(ctx context.Context,
 	return err
 }
 
+func insertOptionalSeedlingParams(ctx context.Context, q PendingAssetStore,
+	batchKey []byte, seedling *tapgarden.Seedling) ([4]sql.NullInt64,
```

Comment:
How about returning a struct instead of an array? That was we can ref each of them by a field instead of the index which might change over time.